File: java_solutions/371.sum-of-two-integers.java
/*
 * @lc app=leetcode id=371 lang=java
 *
 * [371] Sum of Two Integers
 */

import java.math.*;
class Solution {
    public int getSum(int a, int b) {
        return BigInteger.valueOf(a).add(BigInteger.valueOf(b)).intValue();
    }
}

File: kotlin_solutions/371.sum-of-two-integers.kt
/*
 * @lc app=leetcode id=371 lang=kotlin
 *
 * [371] Sum of Two Integers
 */

import java.math.BigInteger class Solution { fun getSum(a: Int, b: Int): Int { return BigInteger.valueOf(a.toLong()) .add(BigInteger.valueOf(b.toLong())) .toInt() } }
